miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditKitDescriptorController.java

```diff
@@ -116,16 +109,14 @@ public ModelAndView setupForm(@PathVariable Long kitDescriptorId, ModelMap model
   }
 
   @RequestMapping(method = RequestMethod.POST)
-  public String processSubmit(@ModelAttribute("kitDescriptor") KitDescriptor kitDescriptor, ModelMap model, SessionStatus session)
+  public ModelAndView processSubmit(@ModelAttribute("kitDescriptor") KitDescriptor kitDescriptor, ModelMap model, SessionStatus session)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Replace this persistent entity with a simple POJO or DTO object. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4684)